CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Removed
 
 ### Fixed
+- Fix NPE in GetResult if "found" field is missing ([#14552](https://github.com/opensearch-project/OpenSearch/pull/14552))
```

Comment:
(Nit) Looks like most of the other entries add at the end.   And I think "handle" is better than "fix" here as we're not really changing behavior, just improving exception type.

Reply:
Okay. Can i push this change in changelog to this PR now?

Reply:
I updated the changelog. Moved this entry to the end and replaced "fix" with "handle"